frontend/src/static/js/components/styles/PlaylistForm.scss

```
@@ -157,21 +157,18 @@
 
 		&::placeholder {
 			/* Chrome, Firefox, Opera, Safari 10.1+ */
-			// color: #fefefe;
-			color: #909090;
+			color: var(--text-subtle);
 			opacity: 1; /* Firefox */
 		}
 
 		&:-ms-input-placeholder {
```

Stylelint (17.11.1)

[error] Vendor-prefixed selector ":-ms-input-placeholder" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)